models/module.js

```diff
@@ -0,0 +1,13 @@
+import connector from '#models/databaseUtil';
+
+const moduleSchema = {
+  modNo: { type: Number, required: true },
+  moduleName: { type: String, required: true },
+  moduleOutcome: { type: String, required: true },
+  moduleContents: [{ type: String, required: true }],
+  hrsPerModule: { type: Number, required: true },
+  cognitiveLevels: [{ type: String, required: true }],
+};
+
+const modules = new connector.model('Module', moduleSchema);
+export default modules;
```

Comment:
Very small change, could you please make modules singular? Simply rename it to moduleModel. We need it in PascalCase for models.
```suggestion
const moduleModel= new connector.model('Module', moduleSchema);
export default moduleModel;
```